extensions/eclipse-che-theia-preferences-provider-extension/src/browser/che-task-schema.ts

```diff
@@ -63,3 +67,14 @@ export const CHE_TASK_SCHEMA: che.TaskJSONSchema = {
     },
     additionalProperties: true
 };
+
+@injectable()
+export class SchemaContribution implements FrontendApplicationContribution {
```

Comment:
It would be better to move this class outside this file. To `che-task-schema-contribution.ts` or smth like that. Cause it's not obvious why an extra class exists in schema description.